src/project.test.ts

```diff
@@ -120,4 +130,147 @@ describe('project', () => {
       });
     });
   });
+  describe('resetUnreleasedPackagesChangelog', () => {
```

Comment:
This goes along with a previous comment, but what do you think about `restoreChangelogsForSkippedPackages`? If so, then it would be good to have this name reflect that:

``` suggestion

  describe('restoreChangelogsForSkippedPackages', () => {
```

or at least reflect the current name:

``` suggestion

  describe('restoreUnreleasedPackagesChangelog', () => {
```